src/cryptoadvance/specter/server_endpoints/controller.py

```diff
@@ -137,3 +139,46 @@ def get_scantxoutset_status():
         "active": app.specter.info["utxorescan"] is not None,
         "progress": app.specter.info["utxorescan"],
     }
+
+
+@app.route("/bitcoin.pdf")
+@login_required
+def get_whitepaper():
+    if not app.specter.info["pruned"]:
```

Comment:
would make sense to first check here if the node is connected and on mainnet so when going via direct URL you could return a proper error message

Reply:
Added in this commit https://github.com/cryptoadvance/specter-desktop/pull/905/commits/8844aaf6f089404e10f3e4773b99205dda03c353